internal/host/host.go

```diff
@@ -701,6 +702,22 @@ func (m *Manager) UpdateMachineConfigPoolName(ctx context.Context, db *gorm.DB,
 	return cdb.Model(common.Host{Host: *h}).Updates(map[string]interface{}{"machine_config_pool_name": machineConfigPoolName, "trigger_monitor_timestamp": time.Now()}).Error
 }
 
+func (m *Manager) UpdateIgnitionEndpointToken(ctx context.Context, db *gorm.DB, h *models.Host, token string) error {
+	hostStatus := swag.StringValue(h.Status)
+	if !funk.ContainsString(hostStatusesBeforeInstallation[:], hostStatus) {
+		return common.NewApiError(http.StatusBadRequest,
+			errors.Errorf("Host is in %s state, host machine config poll can be set only in one of %s states",
```

Comment:
/hold
Update error string `machine config poll` should be `ignition endpoint token`